packages/grafana-data/src/types/featureToggles.gen.ts

```diff
@@ -1,14 +1,18 @@
+// NOTE: This file was auto generated.  DO NOT EDIT DIRECTLY!
+// To change feature flags, edit:
+//  pkg/services/featuremgmt/registry.go
+// Then run tests in:
+//  pkg/services/featuremgmt/toggles_gen_test.go
+
 /**
  * Describes available feature toggles in Grafana. These can be configured via
  * conf/custom.ini to enable features under development or not yet available in
  * stable version.
  *
- * Only enabled values will be returned in this interface
- *
  * @public
  */
 export interface FeatureToggles {
-  [name: string]: boolean | undefined; // support any string value
+  // [name: string]?: boolean; // support any string value
```

Comment:
```suggestion
```
?

Reply:
Ahh -- i just added a comment that we should uncomment this line before merging.  When it is commented out we get type errors when new frontend keys are created (3 in the last week!)

While developing it is nice to have in strict mode, but for release it should allow any string.